aws/resource_aws_default_vpc.go

```diff
@@ -45,16 +45,27 @@ func resourceAwsDefaultVpcCreate(d *schema.ResourceData, meta interface{}) error
 		},
 	}
 
-	resp, err := conn.DescribeVpcs(req)
+	var vpcId string
+	describeResp, err := conn.DescribeVpcs(req)
 	if err != nil {
 		return err
 	}
+	if describeResp.Vpcs == nil || len(describeResp.Vpcs) == 0 {
+		log.Printf("[INFO] No default VPC found in this region")
+	} else {
+		vpcId = aws.StringValue(describeResp.Vpcs[0].VpcId)
+	}
 
-	if resp.Vpcs == nil || len(resp.Vpcs) == 0 {
-		return fmt.Errorf("No default VPC found in this region.")
+	if vpcId == "" {
```

Comment:
Why do this here rather than inside the if statement in line 53?

Reply:
Changed as suggested.